dashboard/bin/refresh_pd_workshop_material_orders

```diff
@@ -2,11 +2,18 @@
 require_relative '../../lib/cdo/only_one'
 require_relative '../config/environment'
 
+# Be more tolerant of retriable errors here than the default 2 attempts
+MAX_ATTEMPTS = 10
+
 def main
-  Pd::WorkshopMaterialOrder.active.find_each do |order|
-    # Be more tolerant of retriable errors here than the default 2 attempts
-    order.refresh max_attempts: 10
-    order.save!
+  # First, place orders
+  Pd::WorkshopMaterialOrder.unordered.find_each do |order|
+    order.place_order max_attempts: MAX_ATTEMPTS
+  end
+
+  # Next, refresh status on all active orders
+  Pd::WorkshopMaterialOrder.successfully_ordered.active.find_each do |order|
```

Comment:
We do [log](https://github.com/code-dot-org/code-dot-org/blob/d2e5b3c54e41250eeab5559b719193333634dc49/dashboard/lib/pd/mimeo_rest_client.rb#L169) each time it retries, but are not analyzing them or gathering metrics. It might be useful to collect these, but I don't know what we could do with them.

Yes, we only [retry](https://github.com/code-dot-org/code-dot-org/blob/d2e5b3c54e41250eeab5559b719193333634dc49/dashboard/lib/pd/mimeo_rest_client.rb#L154-L160) known errors. Other failures are logged to honeybadger. However, the next time the cronjob runs it will retry everything. If those failed calls still call, it will up the instance count on honeybadger. We don't have a concept of a never retry error.